eng/scripts/Create-APIReview.ps1

```diff
@@ -0,0 +1,40 @@
+Param(
+    [Parameter(Mandatory=$True)]
+    [string] $ArtifactName,
+    [Parameter(Mandatory=$True)]
+    [string] $OutPath,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiviewUri,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiKey,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiLabel,
+    [Parameter(Mandatory=$True)]
+    [string] $SourceBranch,
+    [Parameter(Mandatory=$True)]
+    [string] $DefaultBranch,
+    [Parameter(Mandatory=$True)]
+    [string] $ConfigFileDir
+)
+
+Write-Host "$PSScriptRoot"
+. (Join-Path $PSScriptRoot .. common scripts common.ps1)
+$createReviewScript = (Join-Path $PSScriptRoot .. common scripts Create-APIReview.ps1)
+Set-Location $PSScriptRoot
+
+Write-Host "Creating API review artifact for $ArtifactName"
+New-Item -ItemType Directory -Path $OutPath/$ArtifactName -force
+
+$gitroot = Join-Path $PSScriptRoot .. ..
+Write-Host "Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName"
+$cmdLine = & $PSScriptRoot/Get-ApiViewCommandLine.ps1 $gitroot $ArtifactName
+Write-Host "Executing clang++ command:"
+Write-Host $cmdLine
+$cmd, $cmdArgs = $cmdLine -split ' '
+& $cmd $cmdArgs > artifact
```

Comment:
Yes, exactly.

Reply:
Do you think a comment would be helpful, or perhaps doing it a different way?

Reply:
What content is being output? Is it just a log or are we dumping code or what? It might make it a little clearer by maybe naming the artifact file something more meaningful and perhaps a comment.

Reply:
It's a command that needs to be evaluated and run that will generate the ast that apiview expects for c++.

E.g. 

```
clang++ D:\a\_work\1\s\sdk\template\azure-template\inc\azure\template.hpp D:\a\_work\1\s\sdk\template\azure-template\inc\azure\template\dll_import_export.hpp <... more headers ...> -Xclang -ast-dump -I <-I /home/ben/sdk/azure-sdk-for-cpp/sdk/identity/azure-identity/inc> <... more headers ...>
```

Reply:
Seems OK for now. At some point we might want to wrap this up in the Get-ApiViewCommandLine.ps1 (or rename it Get-ApiViewArtifact.ps1) so that there is less magic happening between these commands.

Reply:
I agree. @antkmsft I assume you wrote the script the way you did because it made for an easy hand-off between c++ and engsys work? Or is there other context I may be missing as well?

Reply:
Resolving for now but will follow up.